unit_tests/run_L1_test.sh

```diff
@@ -0,0 +1,53 @@
+# If not stated otherwise in this file or this component's license file the
+# following copyright and licenses apply:
+#
+# Copyright 2021 RDK Management
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+cd ..
+python3.7 -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+pip install --editable .
+cd unit_tests/
+
+echo "1. All"
+echo "2. test_Bundleprocessor_ut"
+
+echo "Select anyone from above"
+echo "ex: All"
+
+
+read n
+
+if [ $n == "All" ]
```

Comment:
If we're going to build a menu-driven interface for running tests, it would be nicer to just write a python file (with a python shebang to make it executable directly like as bash script) with argparse or click - will be more scalable in the future.